aten/src/ATen/templates/DispatchKeyNativeFunctions.h

```diff
@@ -10,4 +10,11 @@ struct ${class_name} {
 ${dispatch_declarations}
 
 };
+
+#define EAGER_REGISTRATION ${eager_registration}
+
+#if !EAGER_REGISTRATION
+extern TORCH_API std::function<void(void)> Register${BackendName}${DispatchKey}Modules;
```

Comment:
wait, why is this marked extern?  I thought that the function is defined in RegisterDispatchKey.cpp either way.

I think you can call it from init_ts_backend with just a TORCH_API macro on it?

Reply:
Its marked `extern` as multiple source files include `LazyNativeFunctions` causing a multiple definition of the same symbol error. Marking it `extern` just makes the symbol known, but does not define it.